### PR TITLE
Career frontend update

### PR DESCRIPTION
**Pull request by dnol777**
The job postings should be good to go. Only thing left is to fix to format the date correctly. Will look into using date.js for it.

### CODE REVIEW

**Comment by hxghhhh** [comment]
@dnol777 back merge 😛

**Comment by hxghhhh** [comment]
@Ssk @dnol777

**Comment by hxghhhh** [comment]
@amahfood and @ssk13 review it

File: modules/core/client/views/lightbox/job.client.view.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="row">
+
+    <div class="col s12">
+        <div class="row">
+            <div class="col s12">
+              <p>Job Description: {{job.content}}</p>
```
Comment (ssk13) [inline modules/core/client/views/lightbox/job.client.view.html:6]: I think this should be job.description

**Comment by ssk13** [comment]
Needs a rebase and one little change to make the description show up, but besides that lgtm

**Comment by amahfood** [comment]
Yeah, I think since the DB was updated with the list of preferred/required qualifications, we need to change a few of the DB references.

**Comment by hxghhhh** [comment]
@amahfood do you mean the schema?? or the angular references in the view??

**Comment by amahfood** [comment]
I meant the schema.  When I open the branch and look at it on chrome,  none of the values appear, I think because the schema was updated so we need to update the variables we are using on the front-end.

**Comment by hxghhhh** [comment]
Is this good to go?? @amahfood  @ssk13

**Comment by amahfood** [comment]
@diegonol were you able to update using the new variables so the job info appears?

**Comment by dnol777** [comment]
Yep! Sorry for the late response, I just saw this I have been super busy at home. The job postings should be showing up correctly now!

**Pull request merged**